src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala

```diff
@@ -121,11 +121,28 @@ object ScalafixInterface {
     ScalafixInterface
   ] = new BlockingCache
 
+  private[scalafix] val defaultLogger: Logger = ConsoleLogger(System.out)
+
+  def fromToolClasspath(
+      scalafixBinaryScalaVersion: String,
+      scalafixDependencies: Seq[ModuleID],
+      scalafixCustomResolvers: Seq[Repository],
+      logger: Logger = defaultLogger
+  ): () => ScalafixInterface =
+    fromToolClasspath(
+      scalafixBinaryScalaVersion = scalafixBinaryScalaVersion,
+      scalafixDependencies = scalafixDependencies,
+      scalafixCustomResolvers = scalafixCustomResolvers,
+      logger = logger,
+      callback = new ScalafixLogger(logger)
+    )
```

Comment:
This class is in an internal package so no need to maintain backward compatibility. A test or two might need to be updated though.
```suggestion
```